app/models/user.js

```diff
@@ -12,6 +12,7 @@ export default Model.extend({
   email: attr(),
   firstName: attr(),
   lastName: attr(),
+  githubAuthToken: attr(),
```

Comment:
I'm not really sure if this is what we need here, but we need **a field** from the API.

`github_id` is an alternative and something that should be safe to render for all users. However, in github integrations, we'd still need to render the auth token for current users, if the client is talking to github directly. If the client will be doing it through our API, then we don't need it client side.

@joshsmith, I think this is worth making an issue for, to discuss. What do you think?